libraries/chain/apply_context.cpp

```diff
@@ -91,16 +91,7 @@ void apply_context::exec_one()
          kv_iterators.resize(1);
          kv_destroyed_iterators.clear();
          if (!context_free) {
-            switch (control.get_config().backing_store) {
-               case backing_store_type::ROCKSDB:
-                  kv_backing_store = create_kv_rocksdb_context(control.kv_database(), control.kv_undo_stack(), receiver, create_kv_resource_manager(*this), control.get_global_properties().kv_configuration);
-                  break;
-               case backing_store_type::NATIVE:
-                  kv_backing_store = create_kv_chainbase_context(db, receiver, create_kv_resource_manager(*this), control.get_global_properties().kv_configuration);
```

Comment:
I would hesitate to move this piece of code away, as "db" is created by apply_context. I don't see combined_session has access to "db". This would break KV on Chainbase.

Reply:
Lin and I discussed this issue over Slack.